contrib/changelog.hbs

```diff
@@ -1,5 +1,6 @@
 {{header}}
 {{toHeader header "-"}}
+Date: {{date}}
```

Comment:
We need a newline before this

Reply:
Im so sorry but I have deleted the branch via which I had made changes. Shall I make a new pull request?

Reply:
I think is better to continue the PR on this branch, so we don't lost the review... Maybe you don't have an option on the bottom like `restore branch` or something?

Reply:
Im sorry but I think I've lost the branch that i worked upon.. its not showing in my repo :cry: are you sure I shall not create a new PR, because its just a single line of code that has been changed :(

Reply:
Can you search if there is a way of restoring your previous branch first? Please. I see that all your PRs were affected. If not, making a new PR makes sense here.

Reply:
@stsewd I am sorry for the mess i've made here. even my PR on issue #3757 have been a mess because of this. I googled the solution but was not able to find any. The main issue was that i deleted the forked repo from my profile (i know thats dumb). Please allow me a new PR and I will keep in mind these things in future. Again, I am so sorry for the inconvenience caused to you :(

Reply:
Don't worry too much about that, it's fine creating a new PR and linking that one here.